charts/elasticsearch/templates/exporter/es-exporter-deployment.yaml

```diff
@@ -66,23 +66,7 @@ spec:
                     "--web.listen-address=:{{ .Values.exporter.service.httpPort }}",
                     "--web.telemetry-path={{ .Values.exporter.web.path }}"]
           securityContext:
-            capabilities:
```

Comment:
where have we moved this for podSecurity capabilities?

Reply:
we have completely removed the hardcoding and moved to its own values dropping ALL privs by default

Reply:
below changes are added to es chart values

```
capabilities:
      drop:
      - ALL
    readOnlyRootFilesystem: true
```